service-commands/src/commands/seed/cliToCommandMap.js

```diff
@@ -0,0 +1,229 @@
+const fs = require('fs')
+const {
+    SeedUtils
+} = require('../../utils')
+
+const {
+  parseMetadataIntoObject,
+  getProgressCallback,
+  getUserProvidedOrRandomTrackFile,
+  getUserProvidedOrRandomImageFile,
+  getUserProvidedOrRandomTrackMetadata,
+  getRandomUserIdFromCurrentSeedSessionCache,
+  getRandomTrackIdFromCurrentSeedSessionCache,
+  addTrackToSeedSessionCache,
+  getActiveUserFromSeedSessionCache,
+  getRandomString
+} = SeedUtils
+
+/*
+This file is the entrypoint for adding new methods.
+
+Each entry must be structured as follows:
+{
+    'some-command': {
+      api: 'LibsClass',
+      description: 'high-level description',
+      method: 'methodName', // belonging to LibsClass
+      params: [
+        {
+          name: 'someParam', // should be camelCased; converted to kebab-case for CLI input automagically
+          description: 'what this param is',
+          userInputHandler: <function>, // optional
+          defaultHandler: <function> // optional
+        }
+      ],
+      onSuccess: <function> // optional, called with response from `await LibsClass.methodName(someParam)`
+    }
+}
+
+^ The above, when added to this file, may be called on the command-line as `A seed some-command --some-param 'userinput'`.
+
+Notes on params:
+  * Params must be in order of original params in fn signature of libs API method.
+  * All commands accept -id or --user-id param to set user that is performing the action.
+  * Each param has an optional userInputHandler which coerces CLI user input to a format that can be accepted by the libs API class. Typically this is a constructor like Boolean/Number or something more involved like parseMetadataIntoObject, which converts comma-separated values into a JS Object.
+  * Each param has an optional defaultHandler which combines/handles user-provided value asynchronously/provides randomized or default value for option (usually from the SeedSession instance, the second provided argument) when user-provided value is unavailable.
+
+*/
+
+const CLI_TO_COMMAND_MAP = {
+    'upload-track': {
```

Comment:
All these methods are working right now. (Adding images does not work.) Try them out!